### DISCUSSION TITLE
custom toolbar handler

### DISCUSSION THREAD

**Discussion by dodgydre** [vueup/vue-quill#270] [General]
Anyway to access the vue instance from the toolbar? In an old project using vue2 I was able to make a custom button on the toolbar that hit an event bus but since vue3 doesn't use that I have been trying to use mitt but 'this' on the toolbar only refers to the toolbar and I can't get back to the vue instance.

**Comment by anburocky3** [comment]
@dodgydre Have you figured out any way?
